Codecademy/gamut
# refactor(ScoreSummary): add optional colorfulIcons prop

### Overview


add optional colorfulIcons prop to ScoreSummary component

also fixed the border which was not showing up in the QuizScore in some cases 
and fixed the height of QuizScore when in row layout and there are more than 5 rows of scores

<img width="861" alt="Screen Shot 2022-12-16 at 2 10 20 PM" src="https://user-images.githubusercontent.com/3805607/208173015-30bc09d8-f4d5-414d-9d9c-9cd9f773e57d.png">

example of use in the LE:
<img width="1399" alt="Screen Shot 2022-12-16 at 2 45 25 PM" src="https://user-images.githubusercontent.com/3805607/208177141-0f3ada82-8811-487a-af1c-f01914babdcf.png">

<img width="1252" alt="Screen Shot 2022-12-19 at 12 16 42 PM" src="https://user-images.githubusercontent.com/3805607/208482978-043a0a3b-bb4e-442c-ac4a-f6b0af551340.png">



### PR Checklist

- [ ] Related to designs: https://www.figma.com/file/XCXshDDbhu0ZHvUDcWFoaB/Assessments-in-a-CJ?node-id=2738%3A56714&viewport=-3382%2C143%2C0.38&t=prDTZSsdSlQ4WegH-0
- [ ] Related to JIRA ticket: [EGG-2966]

#### Testing Instructions
1. visit storybook
2. check out the Colorful Icons story at the bottom and make sure it looks cool
3. Profit!


[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/34033)  | [Monolith Env](https://pr-34033-monolith.dev-eks.codecademy.com/) |
| Portal-app     | [Portal-app PR](https://github.com/codecademy-engineering/mono/pull/911)  | [Portal-app Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-911) |




[EGG-2966]: https://codecademy.atlassian.net/browse/EGG-2966?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
